jekyll/_cci2/server-3-install-build-services.adoc

```diff
@@ -380,6 +380,10 @@ Once you have configured the environmental variables and contexts, rerun the rea
 
 image::realitycheck-pipeline.png[Screenshot showing the realitycheck project building in the CircleCI app]
 
+==== Hosting Build Agent Image in External Image Repository
+
+If you are hosting build agent image (`circleci/picard`), used by Nomad client, in an external or private repo, you have to keep the private image registry as Public and unauthenticated.
```

Comment:
This could probably all go in the Configure and Deploy section.  and it could probably just say something like "consult customer support to set up"

Reply:
moving to configure and deploy.